### DISCUSSION TITLE
HDR10 to DV seems successful but video file always ends up as HDR

### DISCUSSION THREAD

**Discussion by NatyaSadella** [R3S3t9999/DoVi_Scripts#41] [Q&A]
Hi! 

Mode 3 conversion doesn't seem to work anymore for me. A couple of weeks ago I could sucessfully create a DV file from HDR10 input file. But now, all my files end up as HDR instead of DV. 

MediaInfo tells me it's Dolby Vision with correct metadata. 

Where can I start to investigate whats going on now? Nothing has changed on my setup, DoVi Scripts says everything was successfull no errors. 

Please help, I've tried everything now and nothing seems to help.

**Comment by R3S3t9999** [comment]
Hi, which player are you using?  It sounds like a player issue if mediainfo reports DV metadata and dovi_scripts doesn't display any error.
If you can provide a link to a small 2-minute sample of your output, I can have a look.

**Reply by NatyaSadella** [reply]
It's just a 24 second calibration file:

https://www.file-upload.net/download-15398163/test_DV.mp4.html

Media Player is Plex on the latest Apple TV 4K. The file I created a couple of weeks ago is playing fine on this setup and there is pretty much nothing you can do on Mode 3. 

I select Mode 3, then 1, then my file and it's doing it's magic but without DV this time :/

**Comment by R3S3t9999** [comment]
I think you have to use Infuse on the ATV for profile 8 support.  I have some P8 files that will work but most of them wont on PLEX.

**Reply by NatyaSadella** [reply]
Ugh thank you this was the solution, I feel dumb.
